server/testutils/testData.ts

```diff
@@ -161,7 +161,7 @@ export default class TestData {
       courtName: 'Default Court Name',
       status: 'ACTIVE',
       isSentenced: true,
-      date: '2025-01-01',
+      appearanceDate: '2025-01-01',
```

Comment:
had to change this due to another api change that is yet to be implemented in the UI. this fild isnt currently used anywhere